.github/workflows/deploy.yml

```
@@ -0,0 +1,31 @@
+# HMS Networks Solution Center
+# Artifact Deploy Action for this project
+# Version: 1.0
+#
+name: Deploy Assets S3
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0
+
+env:
+  AWS_SECRET_ACCESS_KEY : ${{secrets.AWS_SECRET_ACCESS_KEY}}
+  AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
+  GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+  TARGET_BUCKET: ${{secrets.TARGET_BUCKET}}
+  S3_BASE_URL: ${{secrets.S3_BASE_URL}}
+
+jobs:
+  upload:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: node upload
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - run: yarn && yarn run upload
+        working-directory: "deploy"
```

Add name to section of GitHub action. 

```suggestion
      - name: Run Yarn Upload Script
        run: yarn && yarn run upload
        working-directory: "deploy"
```

---

changed